src/app/content/highlights/components/HighlightStyles.tsx

```diff
@@ -0,0 +1,297 @@
+import React from 'react';
+import styled, { css } from 'styled-components/macro';
+import { InfoCircle } from 'styled-icons/fa-solid/InfoCircle';
+import htmlMessage from '../../../components/htmlMessage';
+import Times from '../../../components/Times';
+import { bodyCopyRegularStyle } from '../../../components/Typography';
+import { H3, h4Style } from '../../../components/Typography/headings';
+import theme from '../../../theme';
+import { toolbarIconColor } from '../../components/constants';
+
+export const desktopPopupWidth = 74.4;
+export const popupPadding = 3.2;
+export const popupBodyPadding = 2.4;
+export const myHighlightsImageWidth = 72.8;
+export const myHighlightsImageHeight = 23.2;
+
+export const stickyNoteMeasures = {
+  blue: 'rgba(13, 192, 220)',
+  bulletSize: 1.6,
+  defaultOffset: 3.2,
+  green: 'rgba(99, 165, 36)',
+  height: 8,
+  left: 32.8,
+  opacity: '0.85',
+  tooltip: {
+    height: 8.8,
+    width: 18,
+  },
+  width: 29.8, /* to allow text to fit in one line with tooltip */
+};
+
+export const imageStyles = css`
+  height: 25.6rem;
+  width: 36rem;
+  box-shadow: 0.1rem 0.1rem 0.4rem 0 rgba(0, 0, 0, 20);
+`;
+
+// tslint:disable-next-line:variable-name
+export const Mask = styled.div`
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  position: fixed;
+  background-color: rgba(0, 0, 0, 0.8);
+`;
+
+// tslint:disable-next-line:variable-name
+export const Modal = styled.div`
+  top: 0;
+  z-index: ${theme.zIndex.modal};
+  left: 0;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  position: fixed;
+  justify-content: center;
+  align-items: center;
+`;
+
+// tslint:disable-next-line:variable-name
+export const Wrapper = styled.div`
+  z-index: 1;
+  width: 100%;
+  background: ${theme.color.neutral.base};
+  margin: 3rem;
+  border-radius: 0.5rem;
+`;
+
+// tslint:disable-next-line:variable-name
+export const Header = styled(H3)`
+  background: #002569;
```

Comment:
this is supposed to match the book color, don't have to change it now but you should make a follow up PR before we forget